internal/services/azuremanagedlustrefilesystem/managed_lustre_file_system_resource.go

```diff
@@ -230,19 +242,16 @@ func (r ManagedLustreFileSystemResource) CustomizeDiff() sdk.ResourceFunc {
 				}
 			}
 
-			_, configSku := metadata.ResourceDiff.GetChange("sku_name")
-			_, configCapacity := metadata.ResourceDiff.GetChange("storage_capacity_in_tb")
+			configSku := metadata.ResourceDiff.Get("sku_name")
+			configCapacity := metadata.ResourceDiff.Get("storage_capacity_in_tb")
+			skuProperties := GetSkuPropertiesByName(configSku.(string))
 
-			if sku := GetSkuPropertiesByName(configSku.(string)); sku != nil {
-				if skuProperties, ok := sku.(SkuProperties); ok {
-					if configCapacity.(int) < skuProperties.MinumumStorage {
-						return fmt.Errorf("'storage_capacity_in_tb' field cannot be less than '%d' for the %q sku, got '%d'", skuProperties.MinumumStorage, configSku, configCapacity)
-					}
+			if configCapacity.(int) < skuProperties.MinimumStorage {
+				return fmt.Errorf("'storage_capacity_in_tb' field cannot be less than '%d' for the %q sku, got '%d'", skuProperties.MinimumStorage, configSku, configCapacity)
+			}
 
-					if configCapacity.(int)%skuProperties.MinumumStorage != 0 {
-						return fmt.Errorf("'storage_capacity_in_tb' field must be defined in increments of '%d' for the %q sku, got '%d'", skuProperties.MinumumStorage, configSku, configCapacity)
-					}
-				}
+			if configCapacity.(int)%skuProperties.MinimumStorage != 0 {
```

Comment:
Also here

Reply:
Why? The field is required and force new, it will never be `nil`?

Reply:
Fixed.